.gitignore

```diff
@@ -14,3 +14,6 @@
 /doxygen_docs
 /doxygen.tag
 /build*
+out
+out
```

Comment:
double? maybe also better to name it `doxygen_cache_out` (I assume this is the name you used for your cache directory).

Reply:
This is part of CLEANUP and will be removed before the final PR

Reply:
Btw out ist the folder VS uses for Cmake projects